### DISCUSSION TITLE
Package postgres_lsp for Nix

### DISCUSSION THREAD

**Discussion by nxy7** [supabase-community/postgres-language-server#50] [Ideas]
Just found out about this LSP while I was fighting some of my sql files. Anyway, I think it might be good idea to package this LSP for Nix, so it's easy to distribute in dev environments. This project is mostly Rust and I think it doesn't have weird build system, so it should not be very hard. If you think it's not ready to be officially distributed flake.nix in this repo would allow anyone to use it in current state.
If you give me green light on this idea I can try packaging it and making PR.

**Comment by psteinroe** [comment]
hey! thanks for opening the discussion. it would be great to have it available in nix. the project is not yet in a usable state, but you can still try to setup the packaging already if you like.
